src/sinks/loki/service.rs

```diff
@@ -84,6 +85,12 @@ impl Finalizable for LokiRequest {
     }
 }
 
+impl MetaDescriptive for LokiRequest {
+    fn get_metadata(&self) -> RequestMetadata {
+        self.metadata
+    }
+}
```

Comment:
This is what is making me ask about a possible wrapper type. AFAICT every one of these functions does exactly this same thing.